lerna.json

```diff
@@ -10,17 +10,10 @@
   "commands": {
     "publish": {
       "exact": true,
-      "scope": [
-        "@raincatcher/angularjs-auth-keycloak",
-        "@raincatcher/angularjs-auth-passport",
-        "@raincatcher/angularjs-http",
-        "@raincatcher/vehicle-inspection",
-        "@raincatcher/angularjs-workflow",
-        "@raincatcher/angularjs-workorder",
-        "@raincatcher/demo-mobile",
-        "@raincatcher/demo-portal",
-        "@raincatcher-examples/step-accident",
-        "@raincatcher-examples/step-vehicle-inspection"
+      "ignore": [
+        "@raincatcher/wfm",
```

Comment:
That may be terrible to maintain as maintainers in core will need to remember to change this here, but I'm ok with this change.

Reply:
Yeah, it's either this flow or the other way around (having to add new angularjs packages here).

Since we only include `core/client/` and `core/common`, I thought using ignore would lead to less need for changes.

Reply:
Simpler will be to just swap config when publishing to not include core locations.